### PR TITLE
Update PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by dstrates**
## Description

Add info to help engineers link issues to Projects

## Linked issues

- (issue)

## Checklist

- [ ] I have added one of the `patch`, `minor`, `major` or `no-release` labels
- [ ] I have performed a self-review of my own code
- [ ] I have checked for redundant or commented out code
- [ ] I have commented my code where I can't make it self-documenting
- [ ] I have made corresponding changes to the documentation
- [ ] I have added any appropriate tests

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Please include a summary of the change with relevant motivation and context
 
 ## Linked issues
 
-- (issue)
+- Link an issue from this repository using the **Development** option
```
Comment (tim-thacker-nullify) [inline PULL_REQUEST_TEMPLATE.md]: this should just be in the checklist

**Review by tim-thacker-nullify — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (dstrates) [reply PULL_REQUEST_TEMPLATE.md]: good call, updated

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -2,13 +2,14 @@
 
 Please include a summary of the change with relevant motivation and context
 
-## Linked issues
+## Linked Issues
 
 - (issue)
```
Comment (tim-thacker-nullify) [inline PULL_REQUEST_TEMPLATE.md:7]: should we just get rid of this section then?

**Review by tim-thacker-nullify — CHANGES_REQUESTED** [review]

File: PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (dstrates) [reply PULL_REQUEST_TEMPLATE.md:7]: Keep it for now because we can't link issues from other repositories

**Review by tim-thacker-nullify — APPROVED** [review]

**Pull request merged**